include/unifex/at_coroutine_exit.hpp

```diff
@@ -313,7 +313,7 @@ inline constexpr struct _fn {
 
 public:
   template(typename Action, typename... Ts)                           //
-      (requires callable<std::decay_t<Action>, std::decay_t<Ts>...>)  //
+      (requires invocable<std::decay_t<Action>, std::decay_t<Ts>...>)  //
```

Comment:
Should this be `std::invocable_v`?
```suggestion
      (requires std::invocable_v<std::decay_t<Action>, std::decay_t<Ts>...>)  //
```

Reply:
It's using our concept shim https://github.com/facebookexperimental/libunifex/blob/d5639a2bb88b8a5375a9c4e8665e0111f2ca336a/include/unifex/detail/concept_macros.hpp#L377-L385

Reply:
Okay. My updates were me trying to get that to work. But I'll just drop it altogether then.